math/cmath/include/algebra/math/impl/cmath_vector.hpp

```diff
@@ -36,18 +119,11 @@ requires std::is_scalar_v<scalar_t> ALGEBRA_HOST_DEVICE inline scalar_t dot(
   return ret;
 }
 
-/// Dot product between two input vectors
-///
-/// @param a the first input vector
-/// @param b the second input matrix with single column
-///
-/// @return the scalar dot product value
```

Comment:
I put the functions between a /// @{ /// @} because they all do the same thing essentially. Or is this not applying a single copy of the comment to all of them?